scripts/test_legacy.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
```

Comment:
Would be better to add a flag to set a variable to the existing script for LEGACY.

Reply:
Done.